angular-ngc/applications/demo/BUILD.bazel

```diff
@@ -5,5 +5,6 @@ package(default_visibility = ["//visibility:public"])
 # The Angular demo application
 ng_application(
     name = "demo",
-    deps = ["//libraries/lib-a"],
+    deps = ["//:lib/lib-a", "//:lib/common"],
```

Comment:
I think this is a bug. The demo app does not directly depend on the `lib/common` npm module (named `@ngc-example/common`). I'm going to try to reproduce this issue...